services/brig/schema/src/V60.hs

```diff
@@ -0,0 +1,41 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE QuasiQuotes #-}
+
+-- This file is part of the Wire Server implementation.
+--
+-- Copyright (C) 2020 Wire Swiss GmbH <opensource@wire.com>
+--
+-- This program is free software: you can redistribute it and/or modify it under
+-- the terms of the GNU Affero General Public License as published by the Free
+-- Software Foundation, either version 3 of the License, or (at your option) any
+-- later version.
+--
+-- This program is distributed in the hope that it will be useful, but WITHOUT
+-- ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+-- details.
+--
+-- You should have received a copy of the GNU Affero General Public License along
+-- with this program. If not, see <https://www.gnu.org/licenses/>.
+
+module V60
+  ( migration,
+  )
+where
+
+import Cassandra.Schema
+import Imports
+import Text.RawString.QQ
+
+migration :: Migration
+migration =
+  Migration 60 "Add team_invitation_email table" $
+    schema'
+      [r|
+        CREATE TABLE team_invitation_email
+            ( email text PRIMARY KEY
+            , team set<uuid>
+            , id uuid
+            , codes set<ascii>
+            )
```

Comment:
Move user to the end?  And it would be slightly nicer to know which team id belongs to which invitation code.  So, perhaps:

```
  ( email text
  , code ascii
  , team uuid
  , user uuid
  ) primary key (email, code)
```

Reply:
A primary key on two elements is a good idea and simplifies the cases; I went for this now:

```
        CREATE TABLE team_invitation_email
            ( email text
            , team uuid
            , invitation uuid
            , code ascii
            , primary key (email, team)
            )
```

Reply:
[Nit-pick] that means two invitations from the same team overwrite each other in chronological order.  I think that's good, but should there be a comment about that somewhere?  (I don't think we should leave comments here in the cql migration code, it's hard to find and may be lost during migration squashing.)